src/posit/connect/urls.py

```diff
@@ -34,12 +45,23 @@ def append_path(url: str, path: str) -> str:
             f"url must be absolute (e.g., http://example.com/__api__): {url}"
         )
 
-    if not (split.path and split.path.endswith("/__api__")):
```

Comment:
Since we're always adding `__api__` before we validate, we'll never raise this error in practice